pkg/tokens/builtin/server/webservice_test.go

```diff
@@ -35,17 +35,15 @@ func (s *staticTokenIssuer) Generate(context.Context, issuer.DataplaneIdentity,
 	return s.resp, nil
 }
 
-type zoneIngressStaticTokenIssuer struct {
-}
+type zoneIngressStaticTokenIssuer struct{}
 
 var _ zoneingress.TokenIssuer = &zoneIngressStaticTokenIssuer{}
 
 func (z *zoneIngressStaticTokenIssuer) Generate(ctx context.Context, identity zoneingress.Identity, validFor time.Duration) (zoneingress.Token, error) {
 	return fmt.Sprintf("token-for-%s", identity.Zone), nil
 }
 
-type zoneStaticTokenIssuer struct {
```

Comment:
How can the formatting change?

Reply:
I was working on another branch as well and I've probably run this using a newer version of the linter and the older version of the linter seems to accept both approaches?